src/main/java/org/interledger/InterledgerPacket.java

```diff
@@ -0,0 +1,245 @@
+package org.interledger;
```

Comment:
I like this idea. Some examples will help a lot to eek out any design tweaks. Perhaps we should update java-ilp-client to use this as a first pass for some implementation experience?